astropy_helpers/commands/build_sphinx.py

```diff
@@ -99,6 +92,7 @@ def initialize_options(self):
         self.open_docs_in_browser = False
         self.warnings_returncode = False
         self.traceback = False
+        self.parallel = None
```

Comment:
do we maybe want some parallelism by default?

Reply:
I'm not sure we want to do that since it's still somewhat experimental.  I'd rather leave it opt in if that's ok? (consistent with tests)